tool/ood-gen/lib/release.ml

```diff
@@ -55,7 +59,7 @@ let all () =
         body_html = Omd.of_string body |> Hilite.Md.transform |> Omd.to_html;
       })
     "releases/"
-  |> List.sort (fun a b -> String.compare a.date b.date)
+  |> List.sort sort_version
   |> List.rev
```

Comment:
```suggestion
  |> List.sort sort_version |> List.rev
```

This just needs to pass the formatter and otherwise looks good.

Reply:
Isn't 
```ocaml
|> List.sort (Fun.flip sort_version)
```
more efficient?